decidim/decidim
# Allow users to be searched by nickname

#### :tophat: What? Why?
When we search for a user, it is only possible to search by name for now, as @andreslucena suggested it would be better to search by name and nickname

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #8658 

#### Testing
* Enter your nickname in the search bar
* See yourself in the results

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](https://github.com/decidim/decidim/blob/develop/CONTRIBUTING.adoc) to this repository.


### :camera: Screenshots
Before:
<img width="1680" alt="Capture d’écran 2021-12-23 à 18 52 57" src="https://user-images.githubusercontent.com/20232956/147580941-5c79cf11-9b95-4d66-92d2-425be2944c39.png">
After: 
<img width="1680" alt="Capture d’écran 2021-12-28 à 16 17 54" src="https://user-images.githubusercontent.com/20232956/147580936-75be7aeb-bb23-47fb-b6ad-59da361149a5.png">


:hearts: Thank you!
